modules/prov/src/main/resources/contexts/nexus/core/provo20130430/v0.1.0.json

```diff
@@ -0,0 +1,25 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    {
+      "generated": {
+        "@id": "prov:generated"
+      },
+      "used": {
+        "@id": "prov:used"
+      },
+      "qualifiedGeneration": {
+        "@id": "prov:qualifiedGeneration"
+      },
+      "wasRevisionOf": {
+        "@id": "prov:wasRevisionOf"
+      },
+      "hadMember": {
+        "@id": "prov:hadMember"
+      },
+      "wasGeneratedBy": {
+        "@id": "prov:wasGeneratedBy"
+      }
+    }
+  ]
+}
```

Comment:
For the ephys provenance template, we also have a prov:activity relationship between StimulusExperiment and TraceGeneration

Reply:
True. We will update the context later.